157002565 remove unused embeddable models

knowuh opened the pull request:
Removes unused embeddable items from portal.

scytacki commented on db/migration-notes.txt:
```diff
@@ -0,0 +1,22 @@
+
+execute "delete from saveable_external_links where embeddable_type like '%Biologica%'"
+execute "delete from page_elements where embeddable_type like '%Biologica%'"
+execute "delete from portal_offering_embeddable_metadata where embeddable_type like '%Biologica%'"
```
Is this file needed anymore?

knowuh commented:
Hello @scytacki  -- I cleaned things up as per your request.

I also rebased from master so its hard to find the fixes,  but they are all in this commit:
2f637cc
